### ISSUE TITLE
Revisions tab: Radio buttons do not work in expected way

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#88]
URL: https://hcdev.openplus.ca/fr/node/1109/revisions
User Agent: Chrome, Firefox 
Language: Both

Issue Description: Radio buttons on the page are not working in expected way. When there are 2 radio buttons for a section/question, they go left and right between the two (with right/left arrows or up/down arrows). But on this page, they take user up and down. The design is not intuitive and is very confusing for keyboard and screen reader users. 
For 2nd row of the table, it announces column 2 radio button, column 3 radio button. This is not correct. It should announce value of each radio button (as mentions in issue #84) and it should be set of 2 radio buttons per row.

Steps to reproduce: Tab with keyboard to the radio buttons on Revisions page and use arrow keys to navigate and select radio buttons. 

Expected results: Each set/pair should come after each row of the table. Radio button should announce with  screen reader what the user is hearing/selecting. 

![image](https://user-images.githubusercontent.com/14280062/129407967-54b67e95-e3e4-4795-b839-05e6deb140a1.png)

**Comment by qanitah** [comment]
They are still not working how a radio button would be. For each set of radio buttons, the user should know what they're selecting. For example, if the user is in Column A, the user goes up and down (left/right) with the arrow keys but they just get the version # only. They won't know the details that are in the first column. Fieldset/legend should be used and each set of radio buttons should be associated correctly. 

Also, the version numbers don't seem very intuitive and screen reader user can get lost. Usually the version numbers go 1, 2, 3 etc.

Here is what the screen reader announces for the following table's 1st 3 rows. If the user moves the arrow in either A or B column, they will not hear information of Column Revision as that is very important.

row 1  Revision  column 1  Revision
A  column 2  A
B  column 3  B
Operations  column 4  Operations
row 2  Revision  column 1  visited  link    2021-10-01 - 16:26   by   link    harry.belizaire
(Draft)
A  column 2  radio button  checked    Version #590293
blank
B  column 3
Operations  column 4  Current revision
row 3  Revision  column 1  link    2021-10-01 - 16:18   by   link    harry.belizaire
(Draft)
A  column 2
B  column 3  radio button  checked    Version #590292
   Operations  column 4   

![image](https://user-images.githubusercontent.com/14280062/136239790-ff5b40a9-fa82-4e8a-9258-12630e0621f5.png)

Also, the information highlighted is not read by the screen reader.

**Comment by echouin** [comment]
From OP: I ﻿would like to close this one. These issues are related to the Diff module. We cannot fix these ourselves, but we can report the issue to the module maintainer.
﻿this is similar to this critical issue #84
